### PR TITLE
rename the 'flow' binary to 'flow_legacy' and set a symbolic link

### PR DESCRIPTION
**Pull request by andlaus**
this is a precursor of merging flow_ebos into the master branch. hopefully, this won't break any existing setups...

please test thoroughly before merging!

### CODE REVIEW

**Comment by andlaus** [comment]
jenins run Norne please

**Comment by akva2** [comment]
jenkins run norne please

File: CMakeLists.txt [hunk 1]
```diff
@@ -127,22 +127,28 @@ endif (NOT EIGEN3_FOUND)
...
     endif()
 
    include (${CMAKE_CURRENT_SOURCE_DIR}/compareECLFiles.cmake)
-endif()
+ endif()
+
+ # create a symbolic link from flow to flow_legacy
+message("create symlink")
+ADD_CUSTOM_TARGET(flow ALL
+	                COMMAND ${CMAKE_COMMAND} -E create_symlink "flow_legacy" "${CMAKE_BINARY_DIR}/bin/flow")
```
Comment (akva2) [inline CMakeLists.txt]: this only deals with built tree's. if you are to do legacy handling we need that symlink installed.

Comment (akva2) [inline CMakeLists.txt]: install(CODE "execute_process(COMMAND ln -sf ${CMAKE_INSTALL_PREFIX}/bin/flow_legacy ${CMAKE_INSTALL_PREFIX}/bin/flow)")

**Comment by andlaus** [comment]
thanks. I added an `install()` statement to `CMakeLists.txt` and I verified that it works. (the `execute_process` stuff was not necessary, though...)

**Comment by akva2** [comment]
Check where that installed link points

**Comment by andlaus** [comment]
it seems to be correct:

```
and@heuristix:~/tmp/blubber > ls -l bin/flow
lrwxrwxrwx 1 and users 11 Nov  4 16:18 bin/flow -> flow_legacy
and@heuristix:~/tmp/blubber > ./bin/flow ~/src/opm-data/norne/NORNE_ATW2013.DATA 
**********************************************************************
*                                                                    *
*                   This is Flow (version 2017.04-pre)               *
*                                                                    *
* Flow is a simulator for fully implicit three-phase black-oil flow, *
*            and is part of OPM. For more information see:           *
*                       http://opm-project.org                       *
*                                                                    *
**********************************************************************
[...]
```

**Comment by akva2** [comment]
Good.

**Comment by andlaus** [comment]
I've just also checked that it works if it is included into the `PATH` envar and start it via `flow` :)

**Comment by atgeirr** [comment]
Tested locally, seems to work perfectly (did not have to delete build dir for example). Merging.

Thanks for the patch!

**Pull request merged**